### DISCUSSION TITLE
Support for Transmission 4?

### DISCUSSION THREAD

**Discussion by bcomnes** [transmission-remote-gui/transgui#1441] [Q&A] [answered]
Transmission 4 came out recently. Has anyone tried this project with a transmission 4 daemon? Does it still work well?

**Comment by username227** [comment]
I'd also be interested in knowing this, even though my NAS (a MyCloud EX2 Ultra) is stuck on 2.94 and I don't know how to manually update it to 4.0.

**Comment by fredo-47** [comment, marked as answer]
I have used transgui with nightlies of transmission-daemon throughout its development, and never encountered any problems.

**Reply by bcomnes** [reply]
Cool, thanks for checking!

**Reply by fredo-47** [reply]
I am only using non-magnet torrents, though.
Latest version seems to have some issues with magnet-torrents, which is being worked on.
Feel free to check the transmission repository for details.

**Comment by lighterowl** [comment]
It will work in general, but you will run into issues if you try to edit trackers for multiple torrents at a time. See https://github.com/xavery/transgui/issues/65 for details.

My fork at https://github.com/xavery/transgui includes a fix for this as of version 5.18.4.f.

**Reply by bcomnes** [reply]
@xavery are you intending/able to upstream that work?

**Reply by lighterowl** [reply]
@bcomnes No. My fork has diverged too far from upstream at this point.

**Reply by bcomnes** [reply]
Ok cool I'll try it out. Thanks!
